include/fc/io/cfile.hpp

```diff
@@ -53,10 +53,26 @@ class cfile {
       if( !_file ) {
          throw std::ios_base::failure( "cfile unable to open: " +  _file_path.generic_string() + " in mode: " + std::string( mode ) );
       }
+
+      fseek(_file.get(), 0L, SEEK_END);
+      _size = ftell(_file.get());
```

Comment:
What if you write to the file after you open it?

Reply:
good catch. thanks.